waf_manually_disabled.go

```diff
@@ -4,7 +4,7 @@
 // Copyright 2016-present Datadog, Inc.
 
 // Manually set datadog.no_waf build tag
-//go:build datadog.no_waf
+//go:build datadog.no_waf && (linux || darwin) && (amd64 || arm64) && !go1.22
```

Comment:
This (as well as the other build tag changes) remove the feature of been able to report multiple compatibility issues at the same time. How about fixing the issues in the tests rather than changing the build tags on the reporters?

Reply:
> remove the feature of been able to report multiple compatibility issues at the same time

That feature never worked because all the variations end up defining some identical symbols, so you can't have all of them in at the same time. So sure, we could instead change code so that it's possible to have those cases enabled at the same time, by de-dupliciating the symbols in question... but then that would be ADDING a feature, and not fixing what was before :)

Reply:
I stand corrected - the code itself worked fine, only the tests were breaking due to conflicting symbols. That is an easy fix actually.